generation/WinSDK/enums.json

```diff
@@ -30873,5 +30881,140 @@
         "field": "bV4V4Compression"
       }
     ]
+  },
+  {
+    "type": "ulong",
+    "name": "SFGAO_FLAGS",
+    "flags": true,
+    "autoPopulate": {
+      "filter": "SFGAO_",
+      "header": "shobjidl_core.h"
+    },
+    "members": [
```

Comment:
Why autopopulate + manual? Is that redundant?

Reply:
I saw that the constant scraper was not parsing those 3 specific values since they didn't already have a value in ILSpy. So I ended up manually adding them as an entry. Is there a better way I can go about this?

Reply:
Looks like they're defined as aliases for constants in another header/namespace, so your approach is ok.